packages/salesforcedx-utils-vscode/test/jest/helpers/paths.test.ts

```diff
@@ -78,4 +81,162 @@ describe('test project paths', () => {
+      expect(projectPaths.apexTestResultsFolder).toBeDefined();
+    });
+
+    it('should return a path to the apex test results folder based on root workspace', () => {
+      const apexTestResultsFolder = projectPaths.apexTestResultsFolder();
+      expect(apexTestResultsFolder).toEqual(path.join(FAKE_WORKSPACE, TOOLS, TEST_RESULTS, APEX));
+    });
+  });
+
+  describe('test apexLanguageServerDatabase', () => {
+    let stateFolderStub: jest.SpyInstance;
+
+    beforeEach(() => {
+      stateFolderStub = jest
+        .spyOn(projectPaths, 'stateFolder')
+        .mockReturnValue(FAKE_WORKSPACE);
+    });
+    it('should be defined', () => {
+      expect(projectPaths.apexLanguageServerDatabase).toBeDefined();
+    });
+
+    it('should return a path to the apex Language Server Database folder based on root workspace', () => {
+      const apexLanguageServerDatabase = projectPaths.apexLanguageServerDatabase();
+      expect(apexLanguageServerDatabase).toEqual(path.join(FAKE_WORKSPACE, TOOLS, APEX_DB));
+    });
+  });
+
+  describe('test lwcTestResultsFolder', () => {
+    let stateFolderStub: jest.SpyInstance;
+
+    beforeEach(() => {
+      stateFolderStub = jest
+        .spyOn(projectPaths, 'stateFolder')
+        .mockReturnValue(FAKE_WORKSPACE);
+    });
+    it('should be defined', () => {
+      expect(projectPaths.lwcTestResultsFolder).toBeDefined();
+    });
+
+    it('should return a path to the lwc Test Results Folder based on root workspace', () => {
+      const lwcTestResultsFolder = projectPaths.lwcTestResultsFolder();
+      expect(lwcTestResultsFolder).toEqual(path.join(FAKE_WORKSPACE, TOOLS, TEST_RESULTS, LWC));
+    });
+  });
+
+  describe('test testResultsFolder', () => {
+    let stateFolderStub: jest.SpyInstance;
+
+    beforeEach(() => {
+      stateFolderStub = jest
+        .spyOn(projectPaths, 'stateFolder')
+        .mockReturnValue(FAKE_WORKSPACE);
+    });
+    it('should be defined', () => {
+      expect(projectPaths.testResultsFolder).toBeDefined();
+    });
+
+    it('should return a path to the  Test Results Folder based on root workspace', () => {
+      const testResultsFolder = projectPaths.testResultsFolder();
+      expect(testResultsFolder).toEqual(path.join(FAKE_WORKSPACE, TOOLS, TEST_RESULTS));
+    });
+  });
+
+  describe('test debugLogsFolder', () => {
+    let stateFolderStub: jest.SpyInstance;
+
+    beforeEach(() => {
+      stateFolderStub = jest
+        .spyOn(projectPaths, 'stateFolder')
+        .mockReturnValue(FAKE_WORKSPACE);
+    });
+    it('should be defined', () => {
+      expect(projectPaths.debugLogsFolder).toBeDefined();
+    });
+
+    it('should return a path to the debug Logs folder based on root workspace', () => {
+      const debugLogsFolder = projectPaths.debugLogsFolder();
+      expect(debugLogsFolder).toEqual(path.join(FAKE_WORKSPACE, TOOLS, DEBUG, LOGS));
+    });
+  });
+
+  describe('test fileExtensionsMatch IN PROGRESS', () => {
```

Comment:
@gbockus-sf also need some help with these... Haven't been able to make them pass🥲

Reply:
I'll push up a commit